tests/test_helper/bats/services.bats

```diff
@@ -0,0 +1,80 @@
+
+load "${ABS_TOP_TEST_DIRNAME}test_helper/setup_teardown/$(basename "${BATS_TEST_FILENAME//.bats/.bash}")"
+
+setup() {
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/common.bash
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/lxd.bash
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/microovn.bash
+    load ${ABS_TOP_TEST_DIRNAME}../.bats/bats-support/load.bash
+    load ${ABS_TOP_TEST_DIRNAME}../.bats/bats-assert/load.bash
+
+    # Ensure TEST_CONTAINERS is populated, otherwise the tests below will
+    # provide false positive results.
+    assert [ -n "$TEST_CONTAINERS" ]
+}
+
+services_register_test_functions() {
+    bats_test_function \
+        --description "Enable already enabled service" \
+        -- enable_already_enabled_service
+    bats_test_function \
+        --description "Enable not existing service" \
+        -- enable_not_existing_service
+    bats_test_function \
+        --description "Disable non existing service" \
+        -- disable_non_existing_service
+    bats_test_function \
+        --description "Disable enabled service " \
+        -- disable_enabled_service
+    bats_test_function \
+        --description "Disable non enabled service" \
+        -- disable_non_enabled_service
+    bats_test_function \
+        --description "Enable disabled service" \
+        -- enable_disabled_service
+}
+
+enable_disabled_service() {
+    for container in $TEST_CONTAINERS; do
```

Comment:
suggestion: I'd say that it's enough to run the operation on single node. Is there a value in executing it on every node?